### PR TITLE
[SCB-2035]Use vertx Json and Pojo to replace own serizalization mechanism

### PR DESCRIPTION
**Pull request by jungan21**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java [hunk 1]
```diff
@@ -41,86 +39,68 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ClientUtil.class);
 
-  private static ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
+  public static ClientUtil INSTANCE = new ClientUtil();
```
Comment (liubao68) [inline service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java:42]: public static field should be final.

File: service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java [hunk 2]
```diff
@@ -41,86 +39,68 @@
...
 
-  private static Microservice microserviceSelf = new Microservice();
+  private static ServerMicroserviceInstance serviceInstanceForHeartbeat = null;
```
Comment (liubao68) [inline service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java:44]: This is a sgingletan class, a static member seems a little bit hard to figure out if it is correct.

File: service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java (hunk 1 above)
Comment (jungan21) [reply service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java:42]: agreed.

File: service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java (hunk 2 above)
Comment (jungan21) [reply service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java]: changed to non-static ....

**Pull request merged**